src/components/chat/DyadMarkdownParser.tsx

```diff
@@ -498,8 +500,30 @@ function renderCustomTag(
       return null;
 
     case "dyad-command":
-      // Don't render anything for dyad-command
-      return null;
+      try {
+        let action;
+        if (attributes.id || attributes.action) {
+          action = {
+            id: attributes.id || attributes.action,
```

Comment:
this feels a bit too indirect (even though it does work). I think you can use attributes.type and map it to `id` and that should work for all the existing cases.